src/js/components/editor/Toolbar.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="toolbar">
+      <button class="toolbar-item" @click="toggleReconstructedText" v-bind:title="$i18n.str('Editor.ToggleReconstructedText')">
+          <i v-show="state.getters.showReconstructedText" class="fa fa-eye" aria-hidden="true"></i>
+          <i v-show="!state.getters.showReconstructedText" class="fa fa-eye-slash" aria-hidden="true"></i>
+      </button>
+      <button class="toolbar-item" @click="$emit('fullscreen')" v-bind:title="$i18n.str('Editor.Fullscreen')">
+          <i class="fa fa-arrows-alt" aria-hidden="true"></i>
+      </button>
+  </div>
+</template>
+
+<script>
+
+import { Store } from 'vuex'
+
+export default {
+  props: {
+    state: {
+      type: Store,
+      required: true
+    }
+  },
+  methods: {
+    toggleReconstructedText() {
+      this.state.getters.showReconstructedText
+        ? this.state.commit('hideReconstructedText')
+        : this.state.commit('showReconstructedText')
```

Comment:
This state is set for the whole SPA, yes?  I wonder if it should be limited to the current pane, since there may in the future be possibility to have other text panes open, and users may want different settings for each.  Also, does the vuex store apply to the browser session?  If users have 4 tabs of the Scrollery open, will setting something in the vuex store then get applied to every tab?

Reply:
In this case, I've added a local Vuex container. That's only applicable (currently) to the Editor. If it should be applied to the whole SPA, we can easily shift that around. I'm anticipating much more use of that local storage to manage inner-Editor state. I'm planning to evaluate if this approach will work before too long though :)

Regarding the 4 tabs question, that's interesting and I don't know. I suspect not, because the reads/writes from localStorage are probably not reactive to data changes. We could consider wiring something like that up, however.

Reply:
Thank you for the clarification, I'm still learning with this stuff.